overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -91,20 +91,43 @@ func addCommand(name, shortHelp, longHelp string, generator func() command) {
 	}
 }
 
+type ForbiddenCommandError struct {
```

Comment:
`// ForbiddenCommandError conveys that a command cannot be invoked in some context`